src/erl.rs

```diff
@@ -1064,6 +1066,22 @@ fn docs_args_call<'a>(
             ..
         } => tuple(std::iter::once(atom(name.to_snake_case())).chain(args.into_iter())),
 
+        TypedExpr::ModuleSelect {
+            constructor: ModuleValueConstructor::Inline,
+            ..
+        }
+        | TypedExpr::Var {
+            constructor:
+                ValueConstructor {
+                    variant: ValueConstructorVariant::Inline,
+                    ..
+                },
+            ..
+        } => args
+            .get(0)
+            .gleam_expect("No arguments for inline type call")
+            .clone(),
```

Comment:
Oh that clone is annoying! Sorry I've given you some not super good advice before when I told you to do this.

Something like this would avoid cloning:

```suggestion
        } => {
            if args.is_empty() {
                crate::error::fatal_compiler_bug("No arguments for inline type call")
            } else {
                args.swap_remove(0)
            }
        }
```